src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,75 @@
 package core.basesyntax;
 
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
 public class WorkWithFile {
+    private static final int OPERATION_NAME = 0;
+    private static final int OPERATION_AMOUNT = 1;
+    private static final String CSV_SEPARATOR = ",";
 
     public void getStatistic(String fromFileName, String toFileName) {
+        String[] result = readFromFile(fromFileName);
+        writeToFile(toFileName, dataProcessing(result));
+    }
+
+    private String[] readFromFile(String fromFileName) {
+        StringBuilder stringBuilder = new StringBuilder();
+        List<String> strings;
+
+        try {
+            strings = Files.readAllLines(Path.of(fromFileName));
+            if (strings.toString().equals("[]")) {
+                return new String[0];
+            }
+        } catch (IOException e) {
+            throw new RuntimeException("Can't open file! " + fromFileName, e);
+        }
+
+        return strings.toArray(new String[0]);
+    }
+
+    private void writeToFile(String toFile, String dataToWriting) {
+        File file = new File(toFile);
+
+        try (BufferedWriter bufferedWriter = new BufferedWriter(new FileWriter(file, false))) {
```

Comment:
```suggestion
        try (BufferedWriter bufferedWriter = new BufferedWriter(new FileWriter(file))) {
```
false will be set by default, we can shorten this code